src/main/java/nextstep/subway/path/domain/PathFinder.java

```diff
@@ -0,0 +1,51 @@
+package nextstep.subway.path.domain;
+
+import nextstep.subway.line.domain.Lines;
+import nextstep.subway.station.domain.Station;
+import nextstep.subway.station.domain.Stations;
+import org.jgrapht.alg.shortestpath.DijkstraShortestPath;
+import org.jgrapht.graph.DefaultWeightedEdge;
+import org.jgrapht.graph.WeightedMultigraph;
+
+import java.util.List;
+import java.util.NoSuchElementException;
+
+public class PathFinder {
+
+    private DijkstraShortestPath dijkstraShortestPath;
```

Comment:
요 부분은 취향 차이겠지만, 아래와 같이 명시적으로 제네릭 타입을 지정해주시지 않으면 `getVertexList`에서 명확한 타입 추론이 되지 않을 것 같아요.

아래와 같이 명확한 제네릭 타입을 남겨주시고, final로 선언해주시는건 어떨까요?
```suggestion
    private final DijkstraShortestPath<Station, DefaultWeightedEdge> dijkstraShortestPath;
```

Reply:
제네릭 타입을 명시하여 수정하였습니다 ^^
그리고 final의 경우는 만약 개발하는 로직에서 초기화 이후 변경이 없다면
final을 지정해주는 버릇을 기르는것이 도움이 될까요?
위의 소스와 같은 경우도 한번 초기화이후 해당 변수의 변동이 없어서 final선언으로 해주는것이 좀더 안정적인 느낌을 받았습니다.

Reply:
final을 붙여주시면, 의도치 않게 중간에 변경되는 경우를 식별할 수 있어 코드 작성에 도움이 될 수 있을것 같아요~